smart-contracts/contracts/BridgeBank/BridgeBank.sol

```diff
@@ -310,26 +282,11 @@ contract BridgeBank is BankStorage,
         string memory _symbol,
         uint256 _amount
     ) public onlyCosmosBridge whenNotPaused {
-        // Confirm that the bank has sufficient locked balances of this token type
-        require(
-            getLockedFunds(_symbol) >= _amount,
-            "!Bank funds"
-        );
+        string memory symbol = safeLowerToUpperTokens(_symbol);
 
         // Confirm that the bank holds sufficient balances to complete the unlock
-        address tokenAddress = lockedTokenList[_symbol];
-        if (tokenAddress == address(0)) {
-            require(
-                ((address(this)).balance) >= _amount,
-                "Insufficient ethereum balance for delivery."
-            );
-        } else {
-            require(
-                BridgeToken(tokenAddress).balanceOf(address(this)) >= _amount,
-                "Insufficient ERC20 token balance for delivery."
-            );
-        }
-        unlockFunds(_recipient, tokenAddress, _symbol, _amount);
+        address tokenAddress = lockedTokenList[symbol];
```

Comment:
@banshee It would be great if we could test these changes if we upgrade the current mainnet deployment with the hardhat setup.